src/Magento/FunctionalTestingFramework/Console/GenerateTestsCommand.php

```diff
@@ -56,13 +70,14 @@ protected function configure()
      *
      * @param InputInterface  $input
      * @param OutputInterface $output
-     * @return void
+     * @return void|integer
      * @throws TestFrameworkException
      * @throws \Magento\FunctionalTestingFramework\Exceptions\TestReferenceException
      * @throws \Magento\FunctionalTestingFramework\Exceptions\XmlException
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        $ioStyle = new SymfonyStyle($input, $output);
```

Comment:
$ioStyle is available in Base Command, just call setOutputStyle() before using it.